conf/nginx/nginx.conf.docs.upstream.template

```diff
@@ -0,0 +1,5 @@
+!{explode server(docs)}
+upstream docs-${server_id}
+{
+  server ${server_hostname}:9980;
```

Comment:
hard coded port number. 
Do we want this to be configurable? 
if so I think It will need changes in ldap schema , zmconfigd and zmproxyconfgen.

Reply:
Yes, i'll be configurable but in the first iteration i would like to avoid any LDAP change since we can affort it, we'll create addictional attributes once we are sure about them.

Reply:
we can affort it = being the docs a standalone server we can ignore customization